common/src/test/java/org/apache/sedona/common/raster/RasterTestBase.java

```diff
@@ -52,6 +52,11 @@ public class RasterTestBase {
     GridCoverage2D oneBandRaster;
     GridCoverage2D multiBandRaster;
     byte[] geoTiff;
+    byte[] testNc;
+    byte[] testBig;
```

Comment:
Please clean up the code and comments here.

Reply:
Done